shared/feature/permissions/src/main/java/com/epmedu/animeal/permissions/presentation/ui/GeolocationPermissionRequestDialog.kt

```diff
@@ -8,7 +8,10 @@ import com.epmedu.animeal.foundation.dialog.AnimealQuestionDialog
 import com.epmedu.animeal.resources.R
 
 @Composable
-fun GeolocationPermissionRequestDialog(onDismiss: () -> Unit) {
+fun GeolocationPermissionRequestDialog(
+    onDismiss: () -> Unit,
```

Comment:
rename to onCancel, or rename onAgree to onConfirm?

Reply:
Done